doc/sphinx/Pacemaker_Explained/local-options.rst

```diff
@@ -526,6 +511,91 @@ whose location varies by OS (most commonly ``/etc/sysconfig/pacemaker`` or
      - Use this TCP port number for :ref:`Pacemaker Remote <pacemaker_remote>`
        node connections. This value must be the same on all nodes.
 
+   * - .. _pcmk_ca_file:
+
+       .. index::
+          pair: node option; PCMK_ca_file
+
+       PCMK_ca_file
+     - :ref:`text <text>`
+     -
+     - The location of a file containing trusted Certificate Authorities, used to
+       verify client or server certificates. This file must be in PEM format and
+       must be readable by Pacemaker daemons (that is, it must allow read permissions
+       to either the |CRM_DAEMON_USER| user or the |CRM_DAEMON_GROUP| group).
```

Comment:
Is this correct? The controller and executor both run as root, and I don't think any other daemons need access